pkg/cedar/core/threat.go

```diff
@@ -0,0 +1,72 @@
+package cedarcore
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/guregu/null/zero"
+
+	"github.com/cmsgov/easi-app/pkg/appcontext"
+	"github.com/cmsgov/easi-app/pkg/models"
+
+	apithreat "github.com/cmsgov/easi-app/pkg/cedar/core/gen/client/threat"
+)
+
+// GetThreat makes a GET call to the /threat endpoint
+func (c *Client) GetThreat(ctx context.Context, cedarSystemID string) ([]*models.CedarThreat, error) {
+	if !c.cedarCoreEnabled(ctx) {
+		appcontext.ZLogger(ctx).Info("CEDAR Core is disabled")
+		return []*models.CedarThreat{}, nil
+	}
+
+	cedarSystem, err := c.GetSystem(ctx, cedarSystemID)
+	if err != nil {
+		return nil, err
+	}
+
+	// Use GetAuthorityToOperate to retrieve ATO for system
+	cedarATO, err := c.GetAuthorityToOperate(ctx, cedarSystem.VersionID)
```

Comment:
Nitpick about variable naming: it might be more clear to name this variable `cedarATOs`, to make it obvious that multiple ATOs can be returned.